src/app/helptext/storage/volumes/manager/manager.ts

```diff
@@ -40,6 +40,20 @@ force_warning: T("The current pool layout is not recommended.\
 Override the following errors?"),
 force_warnings:{
     'diskSizeWarning': T("One or more data vdevs has disks of different sizes.")
-}
+},
+data_vdev_title: T("Data"),
+data_vdev_description: T("Normal vdev type, used for primary storage operations. A ZFS pool will always have at least one DATA vdev."),
+cache_vdev_title: T("Cache"),
+cache_vdev_description: T("ZFS L2ARC read-cache. Can be removed. Optional vdev that can be used with fast devices to accelerate read operations."),
+log_vdev_title: T("Log"),
+log_vdev_description: T("ZFS LOG device. Can be removed. Optional write-cache that can improve speeds of synchronous writes."),
+spare_vdev_title: T("Hot Spare"),
+spare_vdev_description: T("Drive set aside ready to be inserted into DATA pool vdevs when a device has failed."),
+special_vdev_title: T("Metadata"),
+special_vdev_description: T("Special Allocation class, used to create Fusion pools. Optional vdev type\
+ which is used to speed up metadata and small block IO. Cannot be removed from pool once added."),
```

Comment:
`Cannot be removed from pool once added.` is likely inaccurate. @amotin ?

Reply:
Yes, it is inaccurate. They can be removed.  But there are couple bugs in the area, that require more free space on remaining pool then really necessary for removal.  Hopefully users will have much more normal capacity then special to not hit this issue as I did on my tests.  There is a bug for that on OpenZFS github.